app/models/storage_tables/attachment.rb

```diff
@@ -3,6 +3,8 @@
 module StorageTables
   # Base attachment class for all attachments.
   class Attachment < ApplicationRecord
+    include ActiveModel::Dirty
```

Comment:
Deze is nodig om `changes` te kunnen opvragen?

Reply:
Deze is niet langer nodig